site/content/gitops-quickstart/01-apply-gitops.md

```diff
@@ -0,0 +1,260 @@
+  EKS
+
+[aws-account]: https://aws.amazon.com/account/
+[aws-cli]: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
+[aws-iam-authenticator]: https://github.com/kubernetes-sigs/aws-iam-authenticator
+[aws-kubectl]: https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html
+
+### Getting ready for GitOps
+
+The main point of GitOps is to keep everything (config, alerts, dashboards,
+apps, literally everything) in git and use it as a single source of truth.
+To keep your cluster configuration in git, please go ahead and create an
+_empty_ repository. On Github for example, follow [these steps][github-repo].
+
+[github-repo]: https://help.github.com/articles/create-a-repo
+
+## Standing up your cluster
+
+First we follow the [usual steps](/introduction/getting-started/) to stand
+up a cluster on EKS. Please review the list of flags to see if you need to
+tweak it for your purposes. In essence it is going to be a variation of:
+
+```console
+eksctl create cluster
+```
+
+Once it is finished, you should be able to check the cluster contents and
+see some system workloads:
+
+```console
+$ kubectl get pods --all-namespaces
+NAMESPACE     NAME                      READY   STATUS    RESTARTS   AGE
+kube-system   aws-node-cl5t5            1/1     Running   0          1m
+kube-system   aws-node-k96bc            1/1     Running   0          1m
+kube-system   coredns-d5c56458d-wc68z   1/1     Running   0          9m
+kube-system   coredns-d5c56458d-zz8d6   1/1     Running   0          9m
+kube-system   kube-proxy-d577n          1/1     Running   0          3m
+kube-system   kube-proxy-tbmdd          1/1     Running   0          3m
+$
+```
+
+## Applying GitOps
+
+The following command will set up your cluster with the `app-dev` profile,
+the first GitOps Quick Start. All of the config files you need for a
+production-ready cluster will be in the git repo you have provided and
+those components will be deployed to your cluster. When you make changes
+in the configuration they will be reflected on your cluster.
+
+> This is an experimental feature. To enable it, set the environment
+> variable `EKSCTL_EXPERIMENTAL=true`.
+>
+> Experimental features are not stable and their command name and flags
+> may change.
+
+```console
+EKSCTL_EXPERIMENTAL=true eksctl \
+        gitops apply app-dev-profile \
+        git-url=https://github.com/example/my-eks-config
+```
+
+- xxx: What will be installed on the cluster as part of this command
+
+This will set up Flux on your cluster and load GitOps Quick Start config
+files into your repo. It will use templating to add your cluster name and
+region to the configuration so that key cluster components that need those
+values can work (e.g. `alb-ingress`).
+
+## Your GitOps cluster
+
+- xxx: Describe what happens there
+- xxx: How users should check what is running at the end of the process
+  (e.g. get pods, port forward to kube dashboard, grafana, how they get
```

Comment:
We could add some screenshots of all of these things to add a bit of visual intrigue.